.merlin

```diff
@@ -0,0 +1,7 @@
+PKG lwt lwt.unix cohttp cohttp.lwt rpclib rpclib.json rpclib.syntax message_switch async core
```

Comment:
This is a really useful change, just one minor observation: maybe we could list more libraries here, because there seem to be more libraries in the `_oasis` file.